dist-build/lang-model-tck-runner/src/test/java/org/jboss/weld/lang/model/tck/LangModelTckTest.java

```diff
@@ -51,8 +52,11 @@ public static Archive<?> deploy() {
                 // beans.xml with discovery mode "all"
                 .addAsWebInfResource(new BeansXml(BeanDiscoveryMode.ALL), "beans.xml")
                 .addAsServiceProvider(BuildCompatibleExtension.class, LangModelExtension.class)
-                // add this class into the deployment so that it's subject to discovery
-                .addClasses(LangModelVerifier.class);
+                .addClasses(LangModelExtension.class)
+                // add this package into the deployment so that it's subject to discovery, including its dependencies
+                .addPackage(LangModelVerifier.class.getPackage())
```

Comment:
If you add the package, the line `.addClasses(LangModelExtension.class)` is superflous. Or am I missing something?

Reply:
Both are necessary, as they are in different packages: `org.jboss.weld.lang.model.tck.LangModelExtension` while `org.jboss.cdi.lang.model.tck.LangModelVerifier`

Reply:
Oh my bad, I thought its fully copied from Weld where the [package is the same](https://github.com/weld/core/tree/master/lang-model-tck/src/test/java/org/jboss/weld/lang/model/tck) but it is only partial. Maybe we should move them to a shared package that doesn't contain `weld` in it?

Reply:
Sorry, my bad, misread the configuration.